docs/01-app/03-api-reference/04-functions/fetch.mdx

```diff
@@ -72,7 +72,7 @@ Set the cache lifetime of a resource (in seconds).
 > - If an individual `fetch()` request sets a `revalidate` number lower than the [default `revalidate`](/docs/app/api-reference/file-conventions/route-segment-config#revalidate) of a route, the whole route revalidation interval will be decreased.
 > - If two fetch requests with the same URL in the same route have different `revalidate` values, the lower value will be used.
 > - As a convenience, it is not necessary to set the `cache` option if `revalidate` is set to a number.
-> - Conflicting options such as `{ revalidate: 3600, cache: 'no-store' }` will cause an error.
+> - Conflicting options such as `{ revalidate: 3600, cache: 'no-store' }` will result in a warning in the terminal.
```

Comment:
```suggesstion
> - Conflicting options such as `{ revalidate: 3600, cache: 'no-store' }` are not allowed, both will be ignored and, in development mode a warning will be printed to the terminal.
````

Reply:
Makes sense, updated the PR with your suggestion. Thanks 🙌